packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -162,6 +163,8 @@ export default class MeetingRequest extends StatelessWebexPlugin {
         url: deviceUrl,
         // @ts-ignore - config comes from registerPlugin
         deviceType: this.config.meetings.deviceType,
+        // TODO
+        // machineId: this.webex.internal.device.getMachineId(),
```

Comment:
todo?

Reply:
yeah - this one will need a discussion because it will require fingerprinting, this is to mark the API it should follow.